src/libLookup/Lookup.cpp

```diff
@@ -918,6 +925,12 @@ void Lookup::RetrieveTxBlocks(vector<TxBlock>& txBlocks, uint64_t& lowBlockNum,
         m_mediator.m_txBlockChain.GetLastBlock().GetHeader().GetBlockNum();
   }
 
+  if (INIT_BLOCK_NUMBER == highBlockNum) {
+    LOG_GENERAL(INFO,
```

Comment:
Suggest to use WARNING for this kind of error that would lead to abnormal behavior.